config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
@cbeer's point about localizations is good one and I'm not sure I'd want to argue that doing something special outweighs that. I really don't feel that strongly about the time formats, so leave it up to you guys to decide.